lib/runners/processor/checkstyle.rb

```diff
@@ -25,6 +26,7 @@ class Processor::Checkstyle < Processor
     register_config_schema(name: :checkstyle, schema: Schema.runner_config)
 
     DEFAULT_TARGET = ".".freeze
+    CONFIG_FILE_NAME = "checkstyle.xml".freeze
```

Comment:
[suggestion] 

- I think `original_recommended_file_path` is needless because it will be hard for us to grep the file path.
- `CONFIG_FILE_NAME` seems ambiguous to me.

```suggestion
    DEFAULT_CONFIG_FILE = (Pathname(Dir.home) / "sider_recommended_checkstyle.xml").to_path.freeze
```

Reply:
Fixed.